### PR TITLE
Module 1: Hello Microverse: The branch htmlCSS-feature is created with index.html and style.css files. The pull request is to merge htmlCSS-features branch to main

### PR DESCRIPTION
**Pull request by mirwaisfarahi**
…e!" and color for the text in style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<html>
+    <header>
+        <title>Microverse Project</title>
+        <link rel="stylesheet" href="style.css">
+    </header>
+
+    <body>
+        <h1>Hello Microverse!</h1>
+    </body>
```
Comment (m05e5) [inline index.html]: - [ ] Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @mirwaisfarahi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️

## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve
  
  - [ ] Please as part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO
  
  - [ ] I can see you've added the .gitignore file but as part of the requirement you have to add the "test.md" file to it. you can do that by adding _/test.md_ to your git ignore file. Also, you have to delete the test.md file from your git repo. [Here ](https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-a-git-repository)is a way you can do that
  
  - [ ] Your Pull Request should have a summary describing introduced changes in more detail (you can edit it at the top of the PR)  

  - [ ] Please make sure to have a professional README file. This is a general requirement. I will highly recommend following the [microverse template](https://github.com/microverseinc/readme-template)

  - [ ] Also, as part of the requirement you have to make sure all linters are passed(green). Please kindly fix it

_I'm sure you can do it just keep it up_
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/test.md
+node_modules/
+/package-lock.json
```
Comment (chrystalme) [inline .gitignore]: Kindly update this file to properly indicate the file you are ignoring. Example if your file `test.md` is in the folder as .gitignore, there is no need for a `/`. The same goes for `package-lock.json`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<html>
+    <header>
+        <link rel="stylesheet" href="style.css">
+        <h1>Hello Microverse!</h1>
+    </header>
+
+    <body>
+        
+    </body>
+</html>
```
Comment (chrystalme) [inline index.html]: The structure of your index.html file does not look right. Kindly move `<header>` into the `<body>`. I suggest you use the [emmet plugin](https://emmet.io/) if you do not already have one in your code editor. There is an intellisense to help you with basic structure of a HTML page.

**Review by chrystalme — CHANGES_REQUESTED** [review]
Hello @mirwaisfarahi , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes ♻️ 

- [ ] Kindly fix your linters as they are not being checked.
- [ ] Your `main` branch seems to be ahead of the `html-css-feature` branch. Make sure you are working on the feature branch which you would merge to the main when your project is approved.
- [ ] Also, there is conflict in your PR. Kindly fix it.
- [ ] Kindly check the inline comments in your project 👇🏽  for more to check and fix


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @mirwaisfarahi  

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**